src/autoscaler/metricsforwarder/config/config.go

```diff
@@ -0,0 +1,98 @@
+package config
+
+import (
+	"autoscaler/db"
+	"autoscaler/helpers"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+const (
+	DefaultMetronAddress                      = "127.0.0.1:3458"
+	DefaultCacheTTL             time.Duration = 15 * time.Minute
+	DefaultCacheCleanupInterval time.Duration = 6 * time.Hour
+	DefaultPolicyPollerInterval time.Duration = 40 * time.Second
+)
+
+type Config struct {
+	Logging              helpers.LoggingConfig `yaml:"logging"`
+	Server               ServerConfig          `yaml:"server"`
+	MetronAddress        string                `yaml:"metron_address"`
+	LoggregatorConfig    LoggregatorConfig     `yaml:"loggregator"`
+	Db                   DbConfig              `yaml:"db"`
+	CacheTTL             time.Duration         `yaml:"cache_ttl"`
+	CacheCleanupInterval time.Duration         `yaml:"cache_cleanup_interval"`
+	PolicyPollerInterval time.Duration         `yaml:"policy_poller_interval"`
+}
+
+type ServerConfig struct {
+	Port int `yaml:"port"`
+}
+
+var defaultServerConfig = ServerConfig{
+	Port: 6110,
+}
+
+var defaultLoggingConfig = helpers.LoggingConfig{
+	Level: "info",
+}
+
+type LoggingConfig struct {
+	Level string `yaml:"level"`
+}
+
+type LoggregatorConfig struct {
+	CACertFile     string `yaml:"ca_cert"`
+	ClientCertFile string `yaml:"client_cert"`
+	ClientKeyFile  string `yaml:"client_key"`
+}
```

Comment:
Why put MetronAddress as a separated entry?    I propose the following configuration yaml format which is more consistent with the other components. 
loggregator:
      metron_address:  xxxx
      tls:
         ca_cert:
        client_cert:
        client_key: